package.json

```diff
@@ -15,27 +15,27 @@
   "main": "app/addon.js",
   "license": "MIT",
   "dependencies": {
-    "babel-plugin-transform-runtime": "^6.15.0",
-    "babel-plugin-transform-class-properties": "^6.18.0",
-    "babel-preset-latest": "^6.16.0",
-    "bluebird": "^3.4.6",
-    "denali": "0.0.x",
-    "denali-babel": "0.0.x",
-    "denali-eslint": "0.0.x",
-    "lodash": "^4.16.4",
-    "mongodb": "^2.1.19",
-    "mysql": "^2.10.2",
-    "orm": "^3.1.0",
-    "pg": "^4.5.5",
+    "babel-plugin-transform-class-properties": "^6.22.0",
+    "babel-plugin-transform-runtime": "^6.22.0",
+    "babel-preset-latest": "^6.22.0",
+    "bluebird": "^3.4.7",
+    "denali": "^0.0.20",
+    "denali-babel": "^0.0.10",
+    "denali-eslint": "^0.0.11",
```

Comment:
Let's avoid pegging the versions here for denali and the addons listed. They evolve fast enough right now that it's more burdensome to have to update half a dozen packages for each minor tweak.

In case you weren't aware (a journey through some of the denali commit history will reveal how I was not :wink:), the `^` version operator here actually has surprising effects in the 0.0.x range. Even the npmjs.org semver calculator's help section gets it wrong, despite the calculator getting it right.

`^0.0.1` will only match `0.0.1` exactly, unlike `^0.1.1`, which would match every patch version in the `0.1.x` series. As the `node-semver' module puts it:

> Allows changes that do not modify the left-most non-zero digit in the [major, minor, patch] tuple. In other words, this allows patch and minor updates for versions 1.0.0 and above, patch updates for versions 0.X >=0.1.0, and no updates for versions 0.0.X.

https://github.com/npm/node-semver#caret-ranges-123-025-004